pyaedt/generic/farfield_visualization.py

```
@@ -0,0 +1,2187 @@
+            Theta scan angle in degrees. The default is ``0``.
+        title : str, optional
+            Plot title. The default is ``"RectangularPlot"``.
+        quantity_format : str, optional
+            Conversion data function.
+            Available functions are: ``"abs"``, ``"ang"``, ``"dB10"``, ``"dB20"``, ``"deg"``, ``"imag"``, ``"norm"``,
+            and ``"real"``.
+        image_path : str, optional
+            Full path for the image file. The default is ``None``, in which case the file is not exported.
+        levels : int, optional
+            Color map levels. The default is ``64``.
+        show : bool, optional
+            Whether to show the plot. The default is ``True``. If ``False``, the Matplotlib
+            instance of the plot is shown.
+        polar : bool, optional
+            Generate the plot in polar coordinates. The default is ``True``. If ``False``, the plot
+            generated is rectangular.
+        max_theta : float or int, optional
+            Maximum theta angle for plotting. The default is ``180``, which plots the far-field data for
+            all angles. Setting ``max_theta`` to 90 limits the displayed data to the upper
+            hemisphere, that is (0 < theta < 90).
+
+        Returns
+        -------
+        :class:`matplotlib.pyplot.Figure`
+            Matplotlib figure object.
+
+        Examples
+        --------
+        >>> import pyaedt
+        >>> app = pyaedt.Hfss(version="2024.1", design="Antenna")
+        >>> setup_name = "Setup1 : LastAdaptive"
+        >>> frequencies = [77e9]
+        >>> sphere = "3D"
+        >>> data = app.get_antenna_data(frequencies,setup_name,sphere)
+        >>> data.plot_contour()
+
+        """
+        if not title:
+            title = quantity
+
+        data = self.combine_farfield(phi, theta)
+        if quantity not in data:  # pragma: no cover
+            self.__logger.error("Far field quantity is not available.")
+            return False
```

Function signature doesn't match `bool`. Should we raise an exception since we leverage the `pyaedt_function_handler` ?